deploy/charts/harvester/Chart.lock

```diff
@@ -7,10 +7,10 @@ dependencies:
   version: 0.1.0
 - name: harvester-network-controller
   repository: https://charts.harvesterhci.io
-  version: 0.2.1
+  version: 0.2.4
```

Comment:
can we please help to reset this value, it will be bumped in another PR, thanks.

Reply:
Fixed. Thanks!